packages/sdks/src/blocks/image/image.helpers.ts

```diff
@@ -0,0 +1,75 @@
+// Taken from (and modified) the shopify theme script repo
```

Comment:
all this code is copied from the React SDK verbatim

Reply:
https://github.com/BuilderIO/builder/blob/16cbe4646ffd639d2d255affb410e7642cc5a49d/packages/react/src/blocks/Image.tsx#L61-L94